client/src/AdminLoginTest.jsx

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import React, { useState } from "react";
+import AdminLogin from "./AdminLogin";
+
+// ADMIN LOGIN TEST
+const AdminLoginTest = () => {
+  const [token, setToken] = useState(null);
+
+  const handleLogin = async ({ username, password }) => {
+    try {
+      // UPDATE WITH DEPLOYMENT URL WHEN COMPLETED
+      const response = await axios.post("http://localhost:3001/quest/login", {
```

Comment:
Recommend this be rewritten as a dotenv variable to allow others to test on other ports and to enable use of a domain in production with no major code edits.

see following example:
.env
REACT_APP_SERVER_URL=http://localhost:3001

adminLoginTest.jsx 
const response = await fetch(`${process.env.REACT_APP_SERVER_URL}/quest/login`, {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json'
  },
  body: JSON.stringify({
    username,
    password,
  })
});
line 12 could then be rewritten to